### ISSUE TITLE
Changed tagging logic for git flow release finish

### ISSUE THREAD

**Issue by mdomke** [nvie/gitflow#374]
I accidentally installed the 0.4.2-pre version via homebrew today and was confused by the way the tagging works in that version. Previously (version 0.4.1) the tag was created on the master branch, after the release branch has been merged into it. Now it seems that the tag is created on the last commit of the release branch, which seems not to be a good idea for me. Especially if you have a build-system that relies on git tags and creates a release version if HEAD is a tagged commit and a development version if its one of the following commits.
Could someone explain the logic behind this change to me? And with respect to semantic versioning I don't would consider this to be a version bump in the patch-level!

**Comment by andradei** [comment]
+1
I'm my opinion, if `git flow release finish X.X.X` doesn't merge into _master_, that branch will stay useless and empty for the life of the project...

**Comment by andradei** [comment]
+1
0.4.1 is already tagging a commit from _release/x.x.x_ instead of _master_ for me. I'd also like to know the reasoning behind this decision.

**Comment by knox** [comment]
When the tag is placed on the release branch before that gets merged back it appears in `git describe --tags`. On the other hand, when the tag is placed on master branch only these tags cannot be found from e.g. develop.

So from my point of view the change tagging behaviour was just the right thing to do.

**Comment by andradei** [comment]
Makes sense. A new tag on master or a branch should work on homebrew just fine.

**Comment by ghost** [comment]
I cannot follow your arguments, the release is now deploying a version x.y.1 and the corresponding tag is named release/x.y.0. What sense does that make?

**Comment by nvie** [comment]
Closing all issues as I'm archiving this repo. git-flow lives on at [git-flow-next](https://github.com/gittower/git-flow-next) if you're still using it. Thanks for reporting!
